processor/src/main/java/org/mapstruct/ap/internal/model/annotation/BooleanProperty.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.model.annotation;
+
+import java.util.List;
+
+/**
+ * @author Ben Zegveld
+ */
+public class BooleanProperty extends Property {
+
+    private List<Boolean> values;
+
+    public BooleanProperty(String key, List<Boolean> values) {
```

Comment:
`key`=`elementName`

Reply:
`key` will be renamed when I also rename `key` from the earlier comment. I want these 2 names to be in sync.